### DISCUSSION TITLE
Everything reset after Testing New version

### DISCUSSION THREAD

**Discussion by NotGumballer91** [ppy/osu#19280] [Q&A] [answered]
I was trying to run lazer from cmd (to test out the new merged features).

And now the current lazer reset everything. Keybinds, no beatmaps, skins and etc.
Considering the files still has 12GB, It should be safe right? and Is it possible to recover?

[performance-draw.log](https://github.com/ppy/osu/files/9160590/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/9160592/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/9160593/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/9160594/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9160595/updater.log)
[database.log](https://github.com/ppy/osu/files/9160596/database.log)
[network.log](https://github.com/ppy/osu/files/9160597/network.log)
[performance.log](https://github.com/ppy/osu/files/9160598/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/9160599/performance-audio.log)

**Comment by peppy** [comment, marked as answer]
Yes. The game will even tell you how to when you start it up. You should find an old version of client.realm you can restore into place.

**Reply by NotGumballer91** [reply]
![image](https://user-images.githubusercontent.com/64581009/180259945-2ce1fd2a-9d49-436a-ad24-450c983f78f7.png)

I did re-install but it still nothing

**Reply by peppy** [reply]
Ah I misread. It will keep the new database. You will need to stay on `master` until the next release.

**Reply by NotGumballer91** [reply]
Oh okay. Is there any workarounds or I shouldn't attempt anything?

**Reply by peppy** [reply]
There are no workarounds. Realm databases cannot be downgraded.
